nix/hls.nix

```diff
@@ -12,11 +12,6 @@ let
     src = sources.hls-released;
     configureArgs = "--disable-benchmarks";
     modules = [{ enableSeparateDataOutput = true; }];
-    sha256map =
```

Comment:
why this gets removed? reason is not mentioned in the commit message. why was it here in the first place?

Reply:
Some sha256 hashes of dependencies were wrong upstream, and it was needed to patch it like that for the version we were using. It is not needed anymore.